android/fakedapp/src/main/java/com/solana/mobilewalletadapter/fakedapp/MainViewModel.kt

```diff
@@ -432,7 +431,7 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
         var signedMessages: Array<ByteArray>? = null
         try {
             val result = client.signMessages(messages, addresses).get()
-            Log.d(TAG, "Signed message(s): $result")
+            Log.d(TAG, "Signed message base58: ${Base58EncodeUseCase(result.signedPayloads[0])}")
```

Comment:
I'd love to print out all the payloads here. not just the 0'th one.
Perhaps leave the original Log statement (to dump the whole object, in case we add fields to it in the future), followed by iterating through each payload and printint out the base58 signature?